poetry/orb.yaml

```diff
@@ -1,6 +1,27 @@
 version: 2.1
 description: "Tools for running poetry commands."
 
+commands:
+  install:
+    description: >
+      Install poetry. Requires a pre-existing Python 3.x installation.
+      Currently supported package managers: apk, apt-get.
+    steps:
+      - run:
+          name: install curl
+          command: |
+            if which apk >/dev/null; then
+              apk add --no-cache --no-progress curl
+            elif which apt-get >/dev/null; then
+              apt-get update -qy
+              apt-get install -qy curl
+            else
+              echo >&2 "ERROR: could not find supported package manager"
+              exit 1
+            fi
+      - run: curl -sSL https://install.python-poetry.org | python3
```

Comment:
We should add a `version` field potentially if a specific version of poetry is desired.

``` sh
curl -sSL https://install.python-poetry.org | python3 - --version x.x.x
curl -sSL https://install.python-poetry.org | POETRY_VERSION= x.x.x python3
```

You might need to do a conditional statement (this is for my benefit to practice how it's done):

``` sh
step:
      - when:
          condition: <<parameters.poetry_version>>
          steps:
            - run: curl -sSL https://install.python-poetry.org | python3 - --version  <<parameters.poetry_version>>
      - unless:
          condition: <<parameters.custom_checkout>>
          steps:
            - run: curl -sSL https://install.python-poetry.org | python3 - 
workflows:
```

Reply:
Agreed, and done. Also added a flag for preview versions.